cmake/onnxruntime.cmake

```diff
@@ -57,6 +57,7 @@ foreach(f ${ONNXRUNTIME_PROVIDER_NAMES})
   list(APPEND SYMBOL_FILES "${ONNXRUNTIME_ROOT}/core/providers/${f}/symbols.txt")
 endforeach()
 
+if(NOT ${CMAKE_SYSTEM_NAME} MATCHES "AIX")
```

Comment:
Any reason that generated source cannot be used in AIX?

Reply:
> Any reason that generated source cannot be used in AIX?

Currently, I blocked the usage of generated source **generated_source.c**  and linker script **onnxruntime.lds**  file for AIX. 
AIX linker  is having different format for linker script , not the same as  linux(gnu)  linker.
Also, python script **gen_def.py** which generates these source/linkers script file need to support AIX linker script generation. 
We can consider for future enhancement .

Reply:
@snnn
FYI.